### PR TITLE
mcoa: bump logging resources to 6.0

### PR DESCRIPTION
**Pull request by JoaoBraveCoding**
PR with precedence: https://github.com/rhobs/multicluster-observability-addon/pull/63

### CODE REVIEW

File: operators/multiclusterobservability/manifests/base/multicluster-observability-addon/cluster_management_addon.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
       - name: global
         namespace: open-cluster-management-global-set
         configs:
-          - group: logging.openshift.io
+          - group: observability.openshift.io
```
Comment (moadz) [inline operators/multiclusterobservability/manifests/base/multicluster-observability-addon/cluster_management_addon.yaml:38]: Is this change from upstream, or does it distinguish MCOA CR's as separate to the CLF installed CR's?

**Review by moadz — APPROVED** [review]

File: operators/multiclusterobservability/manifests/base/multicluster-observability-addon/cluster_management_addon.yaml (hunk 1 above)
Comment (JoaoBraveCoding) [reply operators/multiclusterobservability/manifests/base/multicluster-observability-addon/cluster_management_addon.yaml:38]: With Logging Operator 6.0 the logging team deprecated the old CRDs and created a new one, it's still called `ClusterLogForwarder` but they belong to a different group.

TL;DR: Upstream, they are still the CR's managed by CLO

**Review by periklis — APPROVED** [review]
/lgtm

**Review by philipgough — APPROVED** [review]
/lgtm
